src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java

```diff
@@ -51,16 +60,31 @@ public CallbackProcessor(CcdApi ccdApi) {
                                     String caseRef,
                                     CaseDetails exceptionRecord) {
         try {
-            attachCase(exceptionRecordJurisdiction, caseRef);
+            attachCase(exceptionRecordJurisdiction, caseRef, exceptionRecord);
             return success();
         } catch (CallbackException e) {
             return createErrorList(e);
         }
     }
 
-    private void attachCase(String exceptionRecordJurisdiction, String caseRef) {
+    private void attachCase(String exceptionRecordJurisdiction,
+                            String caseRef,
+                            CaseDetails exceptionRecord) {
         CaseDetails theCase = ccdApi.getCase(caseRef, exceptionRecordJurisdiction);
-        ccdApi.startAttachScannedDocs(theCase);
+        StartEventResponse event = ccdApi.startAttachScannedDocs(theCase);
+        Map<String, Object> data = insertNewScannedDocument(exceptionRecord.getData(), theCase.getData());
+        ccdApi.attachExceptionRecord(theCase, data, createEventSummary(exceptionRecord, theCase), event);
+    }
+
+    private String createEventSummary(CaseDetails exceptionRecord, CaseDetails theCase) {
+        return format("Attaching exception record(%d) document number:%s to case:%d",
+            exceptionRecord.getId(),
+            getDocumentNumber(exceptionRecord.getData()),
+            theCase.getId());
+    }
+
+    private String getDocumentNumber(Map<String, Object> data) {
```

Comment:
hmm im thinking a Map<String, Object> data wrapper would be better to deal with and could be replaced with a Jackson object at some point.